### PR TITLE
inductor(CPU): add Conv+binary+unary fusion filter

### PR DESCRIPTION
**Pull request by XiaobingSuper**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #90259

For Conv+binary+unary fusion, we only support conv+add+relu, this PR adds a such check to fix TIMM failed models. 
TODO: enable more Conv+binary+unary fusion to improve TIMM models' performance.

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

### CODE REVIEW

File: torch/_inductor/overrides.py [hunk 1]
```diff
@@ -718,6 +718,12 @@ def fuse_unary(gm: torch.fx.GraphModule):
                     computation_node.padding, str
                 ):
                     continue
+                # TODO: support more conv+binary+unary fusion.
+                if type(computation_node) in [
+                    ConvBinary2d,
+                    ConvBinaryInplace2d,
+                ] and type(unary_node) not in [nn.ReLU]:
+                    continue
```
Comment (EikanWang) [inline torch/_inductor/overrides.py:726]: What's the behavior of `F.relu`?

**Review by EikanWang — COMMENTED** [review]
LGTM

File: torch/_inductor/overrides.py (hunk 1 above)
Comment (XiaobingSuper) [reply torch/_inductor/overrides.py:726]: We don't support it now, only support **nn.module** fusion, for function path, we can enable it at the next step.

Comment (EikanWang) [reply torch/_inductor/overrides.py:726]: @XiaobingSuper , I'm okay with this PR to fix TIMM failed model. And let's submit another PR to support `F.ReLU`.

**Review by EikanWang — APPROVED** [review]

**Comment by XiaobingSuper** [comment]
@jansel

**Review by jgong5 — APPROVED** [review]

**Review by jansel — APPROVED** [review]

**Pull request closed without merging**